tests/main/store-state/task.yaml

```diff
@@ -28,13 +28,16 @@ execute: |
     "$TESTSTOOLS"/store-state -h | MATCH "usage: store-state setup-fake-store <DIR>"
     "$TESTSTOOLS"/store-state --help | MATCH "usage: store-state setup-fake-store <DIR>"
 
-    # Setup staging store
-    "$TESTSTOOLS"/store-state setup-staging-store
-    snap info core | MATCH "store-url:.*https://staging-api.snapcraft.io"
-
-    # Teardown staging store
-    "$TESTSTOOLS"/store-state teardown-staging-store
-    snap info core | MATCH "store-url:.*https://snapcraft.io"
+    # Staging store cannot be used with snapd deb from the repository
```

Comment:
Is this really true? Don't we re-exec into a snapd that trusts the staging store?

Reply:
I tried but staging keys are not used when we re-exec into snapd snap.

Reply:
CC @bboozzoo do you have any idea why this would be the case? I think this is a bug somewhere.

Reply:
@sergiocazzolato can you run the test manually and make sure that when you run `SNAPD_DEBUG=1 snap info core` or whether snapd indicates in the logs that is has reexecd into the snap? maybe the condition here should be to check for SNAPD_REEXEC ?